content-repo/gendocs_test.py

```diff
@@ -231,6 +231,20 @@ def test_process_extra_doc(tmp_path, mdx_server):
         assert f.readline().startswith('custom_edit_url: https://github.com/demisto/content-docs/blob/master/content-repo/extra-docs/integrations')
 
 
+def test_process_private_doc(tmp_path, mdx_server):
+    release_file = f'{os.path.dirname(os.path.abspath(__file__))}/.content-bucket/Packs/HelloWorldPremium/Playbooks/' \
```

Comment:
Don't assume this directory exists. The test data you want to test should reside in `content-repo/test_data`.